linux_os/guide/system/software/integrity/crypto/openssl_use_strong_entropy/rule.yml

```diff
@@ -0,0 +1,38 @@
+documentation_complete: true
+
+# TODO: The plan is not to need this for RHEL>=8.4
+prodtype: rhel8
+
+title: 'OpenSSL uses strong entropy source'
+
+description: |-
+    To set up an <tt>openssl</tt> wrapper that adds a <tt>-rand /dev/random</tt> option to the <tt>openssl</tt> invocation,
+    save the following shell snippet to the <tt>/etc/profile.d/cc-config.sh</tt>:
+    <pre>
+    {{{ openssl_strong_entropy_config_file() | indent(4) }}}
+    </pre>
+
+rationale: |-
+    The <tt>openssl</tt> default configuration uses less robust entropy sources for seeding.
+    The referenced script is sourced to every login shell, and it transparently adds an option
+    that enforces strong entropy to every <tt>openssl</tt> invocation,
+    which makes <tt>openssl</tt> more secure by default.
```

Comment:
This feels vague. What is the default entropy source and why is it weak?

Reply:
That's the best I was able to come up with. I have asked @comps for help.

Reply:
I don't think we should differentiate between "less robust" or "weak" or "strong". The default is good enough for any reasonable use case (to the point that credit card companies use it), this "strong entropy source" really just means using a SP800-90A compliant DRBG.

Reply:
Thanks @comps for explanation.